.github/workflows/release-ocb-components.yml

```diff
@@ -0,0 +1,137 @@
+name: Automation - Test and Create Release Tags for OCB components
+
+on:
+  workflow_dispatch:
+    inputs:
+      ocb-module-version:
+        required: true
+        description: Release Tag version (Follows OTel Collector Versioning, like 0.113.0)
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  E2E_TEST_ACCOUNT_ID: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ACCOUNT_ID }}
+  E2E_TEST_ROLE_NAME: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ROLE_NAME }}
+  OCB_MODULE_VERSION: ${{ inputs.ocb-module-version }}
+
+jobs:
+  setup-ocb-test-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: aws-ocb-dev
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ env.E2E_TEST_ACCOUNT_ID }}:role/${{ env.E2E_TEST_ROLE_NAME }}
+          aws-region: us-east-1
+
+      - name: Retrieve account
+        uses: aws-actions/aws-secretsmanager-get-secrets@v2
+        with:
+          secret-ids: |
+            ACCOUNT_ID, otlp-region-account/gamma-us-east-1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/${{ secrets.E2E_TEST_ROLE_NAME }}
```

Comment:
Why do this a 2nd time? Are we using one acct role to assume to another acct role? Curious what the overall setup here is.

Reply:
>Are we using one acct role to assume to another acct role

Yes. We can workaround this by just adding the 2nd's account's role as another secret, but currently we re-use the existing secret of the 1st account to minimize the number of secrets we add to this repo. BTW the 1st account is currently being used to run the e2e test in the release workflow.